bc_obps/reporting/models/reporting_field.py

```diff
@@ -0,0 +1,19 @@
+from registration.models import BaseModel
+from django.db import models
+
+
+class ReportingField(BaseModel):
+    """Conditional Reporting Fields"""
+
+    # No history needed, these elements are immutable
+    field_name = models.CharField(
+        max_length=1000, db_comment="Name of field needed for the related configuration element."
+    )
+    field_type = models.CharField(max_length=1000, db_comment="Type definition for field.")
+
+    def serialize(self) -> "dict[str,str]":
```

Comment:
Apparently mypy doesn't mind quoted type hints 🤷 

```suggestion
    def serialize(self) -> dict[str, str]:
```

Reply:
My editor says this needs to be quoted
![Screenshot from 2024-06-14 15-32-37](https://github.com/bcgov/cas-registration/assets/26367840/a12ed9b4-d941-4b67-a1e5-66090ef08019)

Reply:
weird! I guess it's correct either way